### PR TITLE
fix: adjust styles of the delegate row on the vote page

### PR DESCRIPTION
**Pull request by w3ea**
## Summary

https://app.clickup.com/t/pvcbh1

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx [hunk 1]
```diff
@@ -102,15 +102,15 @@ export const DelegateRow = ({
 		<TableRow>
 			<TableCell
 				variant="start"
-				innerClassName={cn("space-x-4 font-bold border border-r-0 border-transparent", rowColor)}
+				innerClassName={cn("space-x-4 font-bold border-2 border-r-0 border-transparent", rowColor)}
```
Comment (dated) [inline src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx:105]: what's the point of this border? it shifts the avatar to the right.

File: src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx [hunk 2]
```diff
@@ -126,7 +126,7 @@ export const DelegateRow = ({
 			<TableCell
 				variant="end"
 				className="w-40"
-				innerClassName={cn("justify-end border border-l-0 border-transparent", rowColor)}
+				innerClassName={cn("justify-end border-2 border-l-0 border-transparent", rowColor)}
```
Comment (dated) [inline src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx:129]: what's the point of this border? it shifts the button to the left.

File: src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx (hunk 1 above)
Comment (w3ea) [reply src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx:105]: it's in design for dark mode

Screenshot 2021-07-28 160403

Comment (dated) [reply src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx:105]: 👍🏻  didn't think about dark mode.

this is the shift i'm talking about (the third row is aligned correctly)

image

with border there needs to be an additional negative margin on the cells contents.

Comment (w3ea) [reply src/domains/vote/components/DelegateTable/DelegateRow/DelegateRow.tsx]: done :v:

**Pull request merged**